### ISSUE TITLE
forgot-password throwing error

### ISSUE THREAD

**Issue by ScottEAdams** [payloadcms/next-payload#60]
Any ideas what may be happening here?

```
ERROR (payload): TypeError: req.get is not a function
    at forgotPassword (webpack-internal:///(api)/./node_modules/payload/dist/auth/operations/forgotPassword.js:48:126)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async handler (webpack-internal:///(api)/./node_modules/@payloadcms/next-payload/dist/handlers/[collection]/forgot-password.js:27:9)
```

I am doing as follows:

```
const data = {email: 'me@test.com'}
const response = await fetch(`${process.env.NEXT_PUBLIC_CMS_URL}/api/users/forgot-password`, {
			method: 'POST',
			body: JSON.stringify(data),
			headers: {
				'Content-Type': 'application/json',
			},
		})
```

**Comment by revnelson** [comment]
Getting the same error here.

**Comment by tylandavis** [comment]
@ScottEAdams @revnelson I was able to see the same error on my end, and I believe it has to do with not specifying a `serverURL` in `payload.config.ts`. I successfully made a POST request to the `/forgot-password` endpoint when I added a `serverURL`.

These API operations look for a `serverURL` in your config, and if there isn't one, they try to use `req.get('host')` to pull it from the request. For some reason that function isn't available here, and I suspect it's something to do with how Next handles middleware, which we can look into further.

**Comment by revnelson** [comment]
> @ScottEAdams @revnelson I was able to see the same error on my end, and I believe it has to do with not specifying a `serverURL` in `payload.config.ts`. I successfully made a POST request to the `/forgot-password` endpoint when I added a `serverURL`.
> 
> These API operations look for a `serverURL` in your config, and if there isn't one, they try to use `req.get('host')` to pull it from the request. For some reason that function isn't available here, and I suspect it's something to do with how Next handles middleware, which we can look into further.

Thank you so much for getting back to me! After checking on this, I found the config pointing to a wrongly labeled ENV variable. After fixing that, the reset email successfully went through.

**Comment by JarrodMFlesch** [comment]
@ScottEAdams & @revnelson this has been resolved and released under v0.0.36
